src/Agent.Service/Linux/trace.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Done :) :) :) 
Yes, we could do journalctl -u service.name and see all the logs from the service